README.md

```diff
@@ -2,14 +2,14 @@
 
 [a11y-dialog](http://edenspiekermann.github.io/a11y-dialog/) is a lightweight (1.3Kb) yet flexible script to create accessible dialog windows.
 
-✔︎ No dependencies  
-✔︎ Leveraging the native `<dialog>` element  
-✔︎ Closing dialog on overlay click and <kbd>ESC</kbd>  
-✔︎ Toggling `aria-*` attributes  
-✔︎ Trapping and restoring focus  
-✔︎ Firing events  
-✔︎ DOM and JS APIs  
-✔︎ Fast and tiny  
+✔︎ No dependencies
```

Comment:
You’ll need to restore these spaces otherwise you are going to mess up with the rendering of this list.

Reply:
Whelp, that was my IDE. Will do.